pkg/util/util.go

```diff
@@ -57,15 +57,17 @@ func CurrentPackage(dir string) (string, bool) {
 // getGoModPath recursively traverses up the directory path
 // to find the location of go.mod file.
 func getGoModPath(dir string) (string, error) {
-	// fix the case where this function could recursively run
-	// for "." in case go.mod is not found. It can cause the
-	// stack to overflow and run infinitely.
-	if dir == "/" {
-		return "", fmt.Errorf("could not find go.mod")
+	if _, err := os.Stat(dir); err != nil {
+		return "", fmt.Errorf("error trying to find go.mod from directory %s: %w", dir, err)
 	}
 	if _, err := os.Stat(filepath.Join(dir, "go.mod")); err == nil {
 		return dir, nil
 	}
+
+	if dir == "/" {
```

Comment:
nit: to accommodate all path separators, I think `if filepath.Dir(dir) == dir` is a more robust check

Reply:
Fixed